docs/src/linear_advection.md

```diff
@@ -91,10 +91,14 @@ nothing #hide
 ```@example LinearAdvection
 using Plots
 
-plot(x,u0)
-plot!(x, last(sol.u))
+plot(x, u0; label = "u0", xguide = "x", yguide = "u")
+plot!(x, last(sol.u); label = "u")
 ```
 
+### Using sparse matrices
+
+TODO: Some text
```

Comment:
This TODO note is still open?

Reply:
I thought we take care of this once the sparse matrices work as expected.

Reply:
Fair enough 👍